.golangci.yaml

```diff
@@ -0,0 +1,50 @@
+run:
+  # Default timeout is 1m, up to give more room
+  timeout: 4m
+
+linters:
+  enable:
+  - asciicheck
+  - deadcode
+  - depguard
+  - gofumpt
+  - goimports
+  - importas
+  - revive
+  - misspell
+  - stylecheck
+  - tparallel
+  - unconvert
+  - unparam
```

Comment:
Maybe we remove this based on @komish concern about `ctx`?

Reply:
I do like this one. I'd rather turn it off in just a couple of locations rather than disable altogether.